experimental/GModule/Cohomology.jl

```diff
@@ -170,7 +184,8 @@ end
 function fp_group_with_isomorphism(C::GModule)
   #TODO: better for PcGroup!!!
   if !isdefined(C, :F)
-    if order(Group(C)) == 1
+    @show Group(C)
+    if (!isa(Group(C), FPGroup)) && order(Group(C)) == 1
```

Comment:
meta-remark: does `group(C)` also work (with lowercase `g`)? Then we should use that.

Also, use `is_trivial(group(C))` ?